### DISCUSSION TITLE
How to register multiple tag names at a timein opcua server?

### DISCUSSION THREAD

**Discussion by RajuGujjalapati** [FreeOpcUa/python-opcua#1204] [General]
I have a list of tags, Is there any way to register all tags at a time in add_variable() section without looping it?

I checked the docs, I think you were not accepting list of child nodes to register at a time. Am i right?

**Comment by AndreasHeine** [comment]
https://github.com/FreeOpcUa/python-opcua/blob/1cca33dd5fb70858b5e811b08356ca97520bcbaf/opcua/common/node.py#L663

if you look at the function above the answer is No!

**Reply by RajuGujjalapati** [reply]
Okay thanks, The same applies to set_value() method right(Not accepting multiple tags at a time?)

**Reply by AndreasHeine** [reply]
Serverside: Yes

Clientside: No
https://github.com/FreeOpcUa/python-opcua/blob/1cca33dd5fb70858b5e811b08356ca97520bcbaf/opcua/client/client.py#L658
https://github.com/FreeOpcUa/python-opcua/blob/1cca33dd5fb70858b5e811b08356ca97520bcbaf/opcua/client/client.py#L666

but i think its just not implemented at this point but described in https://reference.opcfoundation.org/v104/Core/docs/Part4/5.7.2/ all necessary classes are existing but its not part of the API
